lib/core/src/display/shape/text/content.rs

```diff
@@ -124,10 +124,19 @@ impl TextChange {
     }
 }
 
+
+
 // ============================
 // === TextComponentContent ===
 // ============================
 
+/// A position of character in multiline text.
```

Comment:
A very minor thing, missing "a" - "A position of character in a multiline text."